app/src/test/java/org/mozilla/fenix/settings/creditcards/CreditCardEditorViewTest.kt

```diff
@@ -291,4 +289,48 @@ class CreditCardEditorViewTest {
             )
         }
     }
+
+    @Test
+    fun `GIVEN invalid name on card WHEN the save button is clicked THEN error message is displayed`() {
+        creditCardEditorView.bind(getInitialCreditCardEditorState())
+
+        val billingName = "       "
+        val cardNumber = "2221000000000000"
+        val expiryMonth = 5
+
+        fragmentCreditCardEditorBinding.cardNumberInput.text = cardNumber.toEditable()
+        fragmentCreditCardEditorBinding.nameOnCardInput.text = billingName.toEditable()
+        fragmentCreditCardEditorBinding.expiryMonthDropDown.setSelection(expiryMonth - 1)
+
+        fragmentCreditCardEditorBinding.saveButton.performClick()
+
+        verify {
+            creditCardEditorView.validateForm()
+        }
+
+        assertFalse(creditCardEditorView.validateForm())
+        assertNotNull(fragmentCreditCardEditorBinding.nameOnCardLayout.error)
```

Comment:
I think we can simply the tests by just adding this check to `GIVEN invalid name on card WHEN the save button is clicked THEN interactor is not called` test